tests/unit_tests/analysis/test_es_update.py

```diff
@@ -220,31 +231,31 @@ def test_update_report_with_different_observation_status_from_smoother_update(
         (
             "IES_ENKF",
             [
-                0.5167529669896218,
-                -0.9178847938402281,
-                -0.6299046429604261,
-                -0.1632005925319205,
-                0.0216488942750398,
-                0.07464619425897459,
-                -1.5587692532545538,
-                0.22910522740018124,
-                -0.7171489000139469,
-                0.7287252249699406,
+                0.49758108622091335,
+                -0.9707583934658779,
+                -0.6519965469839432,
+                -0.11707580894540548,
+                -0.019608044356680288,
+                0.05109583740829979,
+                -1.5443581696894637,
+                0.2589327792111955,
+                -0.6606765119887932,
+                0.686502952382529,
             ],
         ),
         (
             "STD_ENKF",
             [
-                1.3040645145742686,
-                -0.8162878122658299,
-                -1.5484856041224397,
-                -1.379896334985399,
-                -0.510970027650022,
-                0.5638868158813687,
-                -2.7669280724377487,
-                1.7160680670028017,
-                -1.2603717378211836,
-                1.2014197463741136,
+                1.5148876946339733,
+                -0.8483395003318406,
+                -1.5965415595585177,
+                -1.3561271002800206,
+                -0.33841535241913645,
+                0.571083832120555,
+                -2.8563199858661372,
+                1.7120013597632608,
+                -1.4397792180408118,
+                1.2442280019343301,
```

Comment:
This is due to ordering of observations? Any way to keep the old order?

Reply:
I think the reason for this is the introduction of "composite" index in gen data, the S matrix is now ordered by the primary key consisting of `[report_step, index]`, and before it would be ordered by a slightly different "syntax". So I think it would involve some hassle to make the order the same, though the order now should be deterministic and extensible to other response types w arbitrary primary keys